src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -768,14 +769,22 @@ IECore::InternedString g_oslSurfaceShaderAttributeName( "osl:surface" );
 IECore::InternedString g_oslShaderAttributeName( "osl:shader" );
 
 IECore::InternedString g_cameraVisibilityAttributeName( "ai:visibility:camera" );
+IECore::InternedString g_cameraVisibilityAutobumpAttributeName( "ai:autobump:visibility:camera" );
```

Comment:
I think this should be `ai:autobump_visibility:camera` (note the central `_`), because we use Arnold's spelling for attribute names.